bin/bump_version.py

```diff
@@ -96,10 +97,9 @@ def bump_version():
     print()
 
     for action in actions:
-        print('{}  {red}{}{off} → {green}{}{off}'.format(
-            *action,
-            red="\u001b[31m", green="\u001b[32m", off="\u001b[0m"
-        ))
+        print(
+            '{}  {red}{}{off} → {green}{}{off}'.format(*action, red="\u001b[31m", green="\u001b[32m", off="\u001b[0m")
+        )
```

Comment:
Again, I don't see how this is better/cleaner than before.

This should be an f-string, anyway, but that's besides the point, I guess.